README.md

```diff
@@ -0,0 +1,187 @@
+- [A terminal](https://code.visualstudio.com/docs/terminal/basics).
+
+### Setup
+
+Clone this repository to your desired folder or download the Zip folder:
+
+```sh
+    "git clone https://github.com/shihabrafio/Meco-Chocolate.git"
+```
+
+- Navigate to the location of the folder in your machine:
+
+```sh
+    cd ./user/
+```
+
+### Install
+
+Install all dependencies:
+
+```sh
+    "npm install"
+```
+
+### Usage
+
+To run the project, follow these instructions:
+
+- After Cloning this repo to your local machine.
+- To get it running on your default browser and local host, run:
+
+```sh
+     " Open the index.html in the project directory with your preferred browser to run the project"
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+- HTML linter errors run:
+
+```
+npx hint .
+```
+
+- CSS linter errors run:
+
+```
+npx stylelint "**/*.{css,scss}"
+```
+
+- JavaScript linter errors run:
+
+```
+npx eslint .
+```
+
+- For unit testing, run:
+
+```
+npm test
+```
+
+### Deployment <a name="deployment"></a>
+
+You can deploy this project using: GitHub Pages,
+
+- I used GitHub Pages to deploy my website.
+- For more information about publishing sources, see "[About GitHub pages](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#publishing-sources-for-github-pages-sites)".
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :bust_in_silhouette: Author <a name="author"></a>
+
+- GitHub: [@shihabrafio](https://github.com/shihabrafio)
+- Twitter: [@shihab_uddin19](https://twitter.com/shihab_uddin19)
+- LinkedIn: [Shihab Uddin Khan](https://www.linkedin.com/in/shihab-uddin-khan-45620a16a/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :telescope: Future Features <a name="future-features"></a>
+
+- [ ] **Implement button to switch sort type (ascending or descending).**
+- [ ] **Implement completed and not-completed routes for completed and non-completed tasks, respectively.**
+- [ ] **Add backend to store tasks.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :handshake: Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :star:️ Show your support <a name="support"></a>
+
+Give a :star:️ if you like this project and how I managed to build it!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :pray: Acknowledgments <a name="acknowledgements"></a>
+
+- To [Cindy Shin](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), Original design.
+- Creative Commons license of the design
+- I would like to thanks all my coding partners at Microverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :memo: License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] your MIT link is broken. please kindly fix it to make your readme professional